apps/client/src/features/event-editor/EventEditor.tsx

```diff
@@ -169,17 +169,18 @@ export default function EventEditor() {
           />
         </div>
         <div className={style.timeSettings}>
-          <label className={style.inputLabel}>Timer Behaviour</label>
+          <label className={style.inputLabel}>End Action</label>
           <Select
             size='sm'
-            name='timerBehaviour'
-            value={event.timerBehaviour}
-            onChange={(event) => handleChange('timerBehaviour', event.target.value)}
+            name='endAction'
+            value={event.endAction}
+            onChange={(event) => handleChange('endAction', event.target.value)}
           >
-            <option value='start-end'>Start to end</option>
-            <option value='duration'>Duration</option>
-            <option value='follow-previous'>Follow previous</option>
-            <option value='start-only'>Start only</option>
+            {Object.keys(EndAction).map((key) => (
+              <option key={key} value={EndAction[key as keyof typeof EndAction]}>
+                {key}
+              </option>
```

Comment:
Good effort.
I wonder if the typing here is a little complicated 

Since we are dealing with 3-4 options I wonder if being explicit about those options has better observability. It also makes the code less magic

Reply:
i wanted to archive this solution because i could't imediately figure out how to do this. But i will agree it seems to be a little overcomplicated for this application. I willr emove it